src/util/sqlhelper.cpp

```diff
@@ -93,7 +93,8 @@ namespace hsql {
     printExpression(expr->expr, numIndent + 1);
     if (expr->expr2 != nullptr) {
         printExpression(expr->expr2, numIndent + 1);
-    } else if (expr->exprList != nullptr) {
+    }
```

Comment:
I changed this because I saw that a CASE statement only printed its 'else' expression and not the cases itself when fiddling. After having a look at #179 and #113, I'm not sure to keep it.

Reply:
Similar to #179, I think we should pay attention to the test coverage for new code and add tests correspondingly. Since #113 has not been addressed yet, I suggest undoing the change and creating an issue for the mentioned CASE statement observation.

Reply:
I created #191 to address this.

Reply:
Thank you